### PR TITLE
Swift Package Manager support without RxSwift dependency

### LINKED ISSUE #1225: Swift Package Manager support
Since Xcode 11 includes Swift Package Manager, it would be nice to support it

### PR DESCRIPTION
**Pull request by martinpucik**
What does this implement/fix? Explain your changes.
---------------------------------------------------
- Added support for Swift Package Manager with updated package description
- Using InpuBarAccessoryView dependency without linking RxSwift
- 👻 

Does this close any currently open issues?
------------------------------------------
- Closes #1225 

Any relevant logs, error output, etc?
-------------------------------------
- N/A

Any other comments?
-------------------
- N/A

Where has this been tested?
---------------------------
**Devices/Simulators:** N/A

**iOS Version:** iOS 11 - 13

**Swift Version:** 5.2

**MessageKit Version:** 3

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    name: "MessageKit",
+    platforms: [.iOS(.v11)],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(name: "MessageKit", targets: ["MessageKit"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/classdojo/InputBarAccessoryView", .branch("dropRxSPM"))
```
Comment (martinpucik) [inline Package.swift:38]: Linking to fork without RxSwift dependency while waiting on merging this to `InputBarAccessoryView`

**Review by Kaspik — COMMENTED** [review]
- needs readme update to inform users about CocoaPods able to have RxSwift extensions where SPM doesn't have this option
- needs changelog update
- let's wait about two/three days and see if IBAV will be merged

**Review by kinoroy — COMMENTED** [review]
We can't merge this until [SE-0271](https://github.com/apple/swift-evolution/blob/master/proposals/0271-package-manager-resources.md) is implemented in Swift 5.3 which will allow SPM to support bundling assets. 
No solid timeline on 5.3 but it was branched from master last week so will probably have a bake-in period of a few months for them to iron it out. 

I recreated the example app repo [here](https://github.com/kinoroy/MessageKitSPMTest) using this branch as the MessageKit dependency (through SPM) and it crashes with
````
Fatal error: MessageKit: Could not create path to the assets bundle.
````

**Comment by Kaspik** [comment]
@kinoroy I don't see AppDelegate.swift in your project after downloading it.

Also, we use it through SPM on our project whole day today without any issue. I'll check your project for sure, but I believe it should work as long as the assets are included only in the framework - you just can't load them in your main project manually.

EDIT: Also sounds are missing.

**Comment by kinoroy** [comment]
@Kaspik My bad, they weren't added into the repo properly but they were there locally when I tested it. Should be fixed now

**Comment by kinoroy** [comment]
@Kaspik Do you use the typing indicator in your project?

**Comment by Kaspik** [comment]
Ohhh, yeah, tested your Example app  it does work for Advanced example etc (because of custom cell layout), but it doesn't work for default / basic one.

Sad story. It's still helpful for users with non-default styling, but yeah we will need to wait for XCode 12.

**Comment by martinpucik** [comment]
Yes, unfortunately there is still no support for assets in SPM package :( 

But, as `.custom` `MessageStyle` is working fine and someone (like me 😹) may find it useful, I'd like to keep this PR and branch open. 

We could then point any questions about SPM to this conversation

**Review by Kaspik — APPROVED** [review]

**Comment by martinpucik** [comment]
@kinoroy @Kaspik I added support Swift Package with assets in Swift 5.3/Xcode 12. 
I'll keep a branch for Swift 5.2 SPM support without assets up `spm-5_2`

**Pull request merged**